.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,60 @@
+name: Feature Request
+description: File a feature request
+title: "[Feature]: "
+labels: ["Feature"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this Feature request report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Suggest an idea for this project ?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+
+  - type: textarea
+    id: screenshot
+    attributes:
+      label: Please provide an example screenshot
+      placeholder: screenshot
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Checklist
+      multiple: true
+      options:
+          - I have searched the existing issues to make sure this is not a duplicate
+          - I have checked the documentation and found no answer
+    validations:
+      required: true
+
+
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you using
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
```

Comment:
@Jayakumar03 , I think we don't need the `browsers` dropdown for the feature request template. please remove it.

Reply:
Done